e2e/resources/workbench-config.ts

```diff
@@ -1,10 +1,11 @@
 import * as fp from 'lodash/fp';
+import { IConfig } from 'types';
 
 const env = process.env.WORKBENCH_ENV || 'test';
 
 const userCredential = {
-  userEmail: process.env.USER_NAME,
-  userPassword: process.env.PASSWORD,
+  USER_NAME: process.env.USER_NAME,
+  PASSWORD: process.env.PASSWORD,
   institutionContactEmail: 'aou-dev-registration@broadinstitute.org',
```

Comment:
should make this one ALLCAPS too for consistency.

Reply:
Done.